src/miniscript/types/extra_props.rs

```diff
@@ -103,6 +103,16 @@ impl Property for ExtData {
         }
     }
 
+    fn from_txtemplate() -> Self {
+        ExtData {
+            pk_cost: 33 + 2,
+            has_verify_form: true,
```

Comment:
`has_verify_form` stands for whether this fragment can be verify-wrapped for free. For example, `CheckSig` to `CheckSigVerify` do not cost an extra op-code. Therefore, this should be false.

Reply:
Ah! I think you're right on this -- I was confused between has_verify_form meaning "is_verify_coercible" and "has_type_V".

Do you think it would be OK to PR changing this variable name to "is_verify_coercible", "is_verify_castable", "can_cast_to_v", "accepts_v_modifier" or something at some point? That's maybe a bit more directly clear meaning.

Reply:
Sure, changes welcome :) . `is_verify_coercible` certainly gives more clear meaning.